web/src/__tests__/prompts.v1.servertest.ts

```diff
@@ -183,68 +185,192 @@ describe("/api/public/prompts API Endpoint", () => {
     expect(fetchedObservations.body.prompt).toBe("prompt-one");
     expect(fetchedObservations.body.version).toBe(1);
     expect(fetchedObservations.body.isActive).toBe(false);
+    expect(fetchedObservations.body.labels).toEqual([]);
     expect(fetchedObservations.body.createdBy).toBe("user-1");
     expect(fetchedObservations.body.config).toEqual({ temperature: 0.1 });
   });
 
   it("should fetch active prompt when multiple exist", async () => {
-    const promptIdOne = uuidv4();
-    const promptIdTwo = uuidv4();
+    // First prompt is activated
+    const prompt1 = await makeAPICall("POST", "/api/public/prompts", {
+      name: "prompt-name",
+      projectId: "7a88fb47-b4e2-43b8-a06c-a5ce950dc53a",
+      prompt: "prompt1",
+      isActive: true,
+      version: 1,
+      config: {
+        temperature: 0.1,
+      },
+      createdBy: "user-1",
+    });
 
-    await prisma.prompt.create({
-      data: {
-        id: promptIdOne,
-        name: "prompt-name",
-        prompt: "prompt",
-        isActive: false,
-        version: 1,
-        config: {
-          temperature: 0.1,
-        },
-        project: {
-          connect: { id: "7a88fb47-b4e2-43b8-a06c-a5ce950dc53a" },
-        },
-        createdBy: "user-1",
+    // Second prompt also activated
+    const prompt2 = await makeAPICall("POST", "/api/public/prompts", {
+      name: "prompt-name",
+      projectId: "7a88fb47-b4e2-43b8-a06c-a5ce950dc53a",
+      prompt: "prompt2",
+      labels: ["production"],
```

Comment:
This here should have no affect, right? Lets remove it.

Reply:
We just discussed this: 
- this has to remain for typing reasons, labels have no impact on API functionality here